### PR TITLE
feat: register dashpay contract

### PR DESCRIPTION
**Pull request by Alex-Werner**
## Issue being fixed or feature implemented

This PR allow to register dashpay contract on init script execution.

## What was done?
- added dashpay-contract as a dependency
- register a identity for dashpay contract
- register dashpay contract

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

**Comment by shumkov** [comment]
I would postpone it until we complete DashPay DIP and did’t update the schema.

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
   "dependencies": {
     "@dashevo/dashcore-lib": "~0.18.11",
     "@dashevo/dashd-rpc": "~2.0.0",
+    "@dashevo/dashpay-contract": "github:dashevo/dashpay-contract",
```
Comment (shumkov) [inline package.json:53]: Should we a specific version?

Comment (Alex-Werner) [reply package.json]: Done :)

**Review by shumkov — APPROVED** [review]
Good! 👍

**Pull request merged**